modules/programs/autorandr.nix

```diff
@@ -177,6 +177,18 @@ let
         default = null;
         example = "nearest";
       };
+
+      extraConfig = mkOption {
+        type = types.nullOr types.lines;
```

Comment:
This can be just `types.lines` with default value empty string `""`.
Then can just append it to the generated config file unconditionally.